lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -0,0 +1,201 @@
+    <br><br>
+
+
+    %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
+    <h5 class="hd hd-5">${_("Task Status")}</h5>
+    <label for="entrance-exam-task-history">
+        ${_("Show the status for the rescoring tasks that you submitted for this learner and entrance exam.")}
+    </label>
+    <br>
+    <p><input type="button" name="entrance-exam-task-history" value="${_('Show Task Status')}" data-endpoint="${ section_data['list_entrace_exam_instructor_tasks_url'] }" aria-describedby="entrance-exam-task-history-help"></p>
+    <div class="entrance-exam-task-history-table"></div>
+    %endif
+    <hr>
+</div>
+%endif
+
+%endif
+
+%if section_data['access']['instructor']:
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
+<div class="course-specific-container action-type-container">
+    <h4 class="hd hd-4">${_("Adjust all enrolled learners' grades for a specific problem")}</h4>
+    <div class="request-response-error"></div>
+
+    <label for="problem-select-all">
+        ${_("Location of problem in course *")}<br>
+        <span class="location-example">${_("Example")}: block-v1:edX+DemoX+2015+type@problem+block@618c5933b8b544e4a4cc103d3e508378</span>
+    </label>
+    <br>
+    <input type="text" name="problem-select-all" placeholder="${_('Problem location')}">
+    <br><br><br>
+
+    <h5 class="hd hd-5">${_("Attempts")}</h5>
+    <label for="reset-attempts-all">${_("Allows all learners to work on the problem again.")}</label>
+    <br>
+    <input type="button" class="molly-guard" name="reset-attempts-all" value="${_('Reset Attempts to Zero')}" data-endpoint="${ section_data['reset_student_attempts_url'] }">
+    <br><br>
+
+    <h5 class="hd hd-5">${_("Rescore")}</h5>
+    <label for="rescore-actions-all">${_("Rescore submitted responses. The 'Rescore Only If Scores Improve' option updates a learner's score only if it improves in the learner's favor.")}</label>
+    <br>
+    <span name="rescore-actions-all">
+        <input type="button" class="molly-guard" name="rescore-problem-all" value="${_('Rescore All Learners\' Submissions')}" data-endpoint="${ section_data['rescore_problem_url'] }">
+        <input type="button" class="molly-guard" name="rescore-problem-all-if-higher" value="${_('Rescore Only If Scores Improve')}" data-endpoint="${ section_data['rescore_problem_url'] }">
+    </span>
+    <br><br>
+
+    <h5 class="hd hd-5">${_("Task Status")}</h5>
+    <label for="task-history-all">${_("Show the status for the tasks that you submitted for this problem.")}</label>
+    <br>
+    <input type="button" name="task-history-all" value="${_('Show Task Status')}" data-endpoint="${ section_data['list_instructor_tasks_url'] }" aria-describedby="task-history-all-help">
+    <div class="task-history-all-table"></div>
+    <hr>
+</div>
+%endif
+
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
```

Comment:
this if statement is redundant upper one and current one could be joined together.